test/cli/install/security-scanner-pty.py

```
@@ -0,0 +1,162 @@
+                            ready = select.select([master_fd], [], [], 0.1)[0]
+                            if ready:
+                                data = os.read(master_fd, 4096)
+                                if data:
+                                    output += data
+                                    sys.stdout.buffer.write(data)
+                                    sys.stdout.buffer.flush()
+                                else:
+                                    break
+                            else:
+                                break
+                    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)